### ISSUE TITLE
Requirements

### ISSUE THREAD

**Issue by ways** [spslater/borgapi#17]
Hi

Requirements are listed as:

    borgbackup: 1.1.17
    python-dotenv: 0.17.1

But when I try to install it asks for fuse. Is that correct?

`$ pip install borgapi
Collecting borgapi
  Using cached borgapi-0.5.1-py3-none-any.whl (30 kB)
Requirement already satisfied: borgbackup[fuse]==1.1.17 in ./.venv/lib/python3.9/site-packages (from borgapi->-r requirements.txt (line 1)) (1.1.17)
Collecting python-dotenv>=0.17.1
  Using cached python_dotenv-0.19.0-py2.py3-none-any.whl (17 kB)
Requirement already satisfied: packaging in ./.venv/lib/python3.9/site-packages (from borgbackup[fuse]==1.1.17->borgapi->-r requirements.txt (line 1)) (21.0)
Collecting llfuse>=1.3.4
  Using cached llfuse-1.4.1.tar.gz (774 kB)
    ERROR: Command errored out with exit status 2:
     command: /home/larsfp/Downloads/git/borgfrag/.venv/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-g4us2xwu/llfuse_0d618ef5987a4ac5bba3b0724494d954/setup.py'"'"'; __file__='"'"'/tmp/pip-install-g4us2xwu/llfuse_0d618ef5987a4ac5bba3b0724494d954/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-5e05cjl4
         cwd: /tmp/pip-install-g4us2xwu/llfuse_0d618ef5987a4ac5bba3b0724494d954/
    Complete output (4 lines):
    Package fuse was not found in the pkg-config search path.
    Perhaps you should add the directory containing `fuse.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'fuse' found
    ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/b1/d4/44443fbaac6d5b878da99e7c0948ee93c7934fa3b00e48c5363823b583d0/llfuse-1.4.1.tar.gz#sha256=c29c79d96a5aeab51608cae12594a1bf83576d86232f97341c7f779d413a4ec9 (from https://pypi.org/simple/llfuse/) (requires-python:>=3.5). Command errored out with exit status 2: python setup.py egg_info Check the logs for full command output.
  Using cached llfuse-1.4.0.tar.gz (478 kB)
    ERROR: Command errored out with exit status 2:
     command: /home/larsfp/Downloads/git/borgfrag/.venv/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-g4us2xwu/llfuse_7834839a4b014859a3a9b268e02c1236/setup.py'"'"'; __file__='"'"'/tmp/pip-install-g4us2xwu/llfuse_7834839a4b014859a3a9b268e02c1236/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-0cyjmwr_
         cwd: /tmp/pip-install-g4us2xwu/llfuse_7834839a4b014859a3a9b268e02c1236/
    Complete output (4 lines):
    Package fuse was not found in the pkg-config search path.
    Perhaps you should add the directory containing `fuse.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'fuse' found
    ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/5d/3d/970c3fb9fbccfb7338f4b4d9fc1f769f7c886a8821ccc76f4d9c8d437606/llfuse-1.4.0.tar.gz#sha256=cb5dc6052bdd9c19dd15d44cd2876520007029a7733e7b9f0c2022c768e3ca52 (from https://pypi.org/simple/llfuse/) (requires-python:>=3.5). Command errored out with exit status 2: python setup.py egg_info Check the logs for full command output.
  Using cached llfuse-1.3.8.tar.gz (481 kB)
ERROR: No .egg-info directory found in /tmp/pip-pip-egg-info-d_5c2use
`

Ubuntu 21.04

**Comment by spslater** [comment]
> But when I try to install it asks for fuse. Is that correct? 

Yeah, I should've made those requirements a bit clearer. Fuse is needed for mounting and unmounting the repositories / archives (with the `mount` and `umount` commands).

I need to add the ability to install without `fuse` and just disable mount and umount.

I'll take a look at the output closer when I get a chance.

**Comment by spslater** [comment]
I'm not sure if you've ever solved this, but I believe the issue is some missing prerequisites that borgbackup needs prior to installing it.

In the borgbackup installation docs https://borgbackup.readthedocs.io/en/stable/installation.html:
```
sudo apt-get install libfuse-dev fuse    # needed for llfuse
sudo apt-get install libfuse3-dev fuse3  # needed for pyfuse3
```

I will update the README so that it mentions needing to have all of the borgbackup prerequisites installed.

**Comment by spslater** [comment]
No activity in a long time, if issue crops up again, please open a new issue.
